### PR TITLE
fix sso login success message

### PR DESCRIPTION
**Pull request by enahum**
#### Summary
Adds a transitional screen when the SSO is successful

#### Ticket Link
https://hub.mattermost.com/private-core/pl/55xw79opufn6jrzawkuzrrxjtc

#### Checklist
- [ ] Added or updated unit tests (required for all new features)
- [x] Has UI changes
- [ ] Includes text changes and localization file updates
- [ ] Have tested against the 5 core themes to ensure consistency between them.
- [ ] Have run E2E tests by adding label `E2E iOS tests for PR`.

#### Screenshots
image

#### Release Note
```release-note
Added a new message to SSO login screen when the login is successful.
```

### CODE REVIEW

File: app/products/calls/types/calls.ts [hunk 1]
```diff
@@ -151,6 +151,7 @@ export type CallsConnection = {
 
 export type CallsConfigState = CallsConfig & {
     AllowEnableCalls: boolean;
+    GroupCallsAllowed: boolean;
```
Comment (jwilander) [inline app/products/calls/types/calls.ts:154]: This is unrelated, right?

**Review by jwilander — APPROVED** [review]
LGTM 👍

File: app/products/calls/types/calls.ts (hunk 1 above)
Comment (jwilander) [reply app/products/calls/types/calls.ts:154]: Nevermind, I see your other comment

**Review by matthewbirtch — APPROVED** [review]
Thanks @enahum!

File: app/screens/sso/sso_authentication.tsx [hunk 2]
```diff
@@ -87,9 +90,10 @@ const SSOAuthentication = ({doSSOLogin, loginError, loginUrl, serverUrl, setLogi
             const bearerToken = resultUrl.query?.MMAUTHTOKEN;
             const csrfToken = resultUrl.query?.MMCSRF;
             if (bearerToken && csrfToken) {
+                setLoginSuccess(true);
                 doSSOLogin(bearerToken, csrfToken);
             }
-        } else if (Platform.OS === 'ios' || result.type === 'dismiss') {
+        } else if (Platform.OS === 'ios' && result.type === 'dismiss') {
```
Comment (rahimrahman) [inline app/screens/sso/sso_authentication.tsx:96]: @enahum if on Android, and user "dismiss" or (x) out of Okta sso, would this if logic change miss that user?

Comment (enahum) [reply app/screens/sso/sso_authentication.tsx]: @rahimrahman yes thank you, this was unintended.

**Review by rahimrahman — APPROVED** [review]

**Comment by lindalumitchell** [comment]
For future traceability: https://mattermost.atlassian.net/browse/MM-60293
